### PR TITLE
Newpipelinetest

### PR DESCRIPTION
**Pull request by SueSu01**
## Bug

Fixes:

Regression? Last working version:

## PR Checklist

- [ ] PR has a meaningful title
- [ ] PR has a linked issue.
- [ ] Described changes

- **Tests**
  - [ ] Automated tests added
  - **OR**
  
  - [ ] Test exception
  - **OR**
  - [ ] N/A 

- **Documentation**
  
  - [ ] Documentation PR or issue filled
  - **OR**
  - [ ] N/A

### CODE REVIEW

File: eng/pipelines/templates/Apex_Tests_On_Windows_Daily.yml [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+    type: number
+  - name: testMachineCleanUpStrategy
+    type: string
+    default: delete
+    values:
+      - delete
+      - stop
+  - name: isOfficialBuild
+    type: boolean
+    default: false
```
Comment (CiciLi1) [inline eng/pipelines/templates/Apex_Tests_On_Windows_Daily.yml:16]:
```suggestion
    default: true
```

**Comment by CiciLi1** [comment]
Hi @SueSu01, test for your email!

**Comment by CiciLi1** [comment]
Hi @SueSu01, this is a test comment.

**Comment by SueSu01** [comment]
> Hi @SueSu01, test.

@CiciLi1 OK

File: eng/pipelines/templates/Apex_Tests_On_Windows_Daily.yml (hunk 1 above)
Comment (SueSu01) [reply eng/pipelines/templates/Apex_Tests_On_Windows_Daily.yml:16]: > 

thanks.

**Pull request closed without merging**